### DISCUSSION TITLE
Django site fails to serve css when DEBUG=False

### DISCUSSION THREAD

**Discussion by peterbrorsson** [vercel/community#1509] [Help]
### Summary

I have deployed a Django project on Vercel and it works fine as long as I run it with DEBUG=True in the settings but if I change that to False, any static file like CSS or images are no longer served. If I inspect the code in my browser, it is the same paths for these regardless of DEBUG value.

There were a bunch of additional changes to the project to get it from the original Django tutorial form working with the Django 'runserver' into something that worked after Vercel deployment. So, it came as a surprise that just changing the DEBUG to False broke the site completely.

I suspect it is some kind of change needed in settngs.py regarding the STATICS_XXX variables but I have no real clue what I would make a guess on.

My hope is that someone else has struggled with the same issue and found a solution.

**Comment by codewitgabi** [comment]
Did you use whitenoise?
